src/lib/validation/validate.ts

```diff
@@ -74,6 +74,12 @@ export const getFormErrors = async (
   }
 };
 
+function isScalarValue(obj: IValues | Value | Values): obj is Value {
+  if (Array.isArray(obj)) return false;
```

Comment:
This seems redundant as typeof [] is object?

Reply:
Yes, I had the logic the other way around first and forgot to clean this up.

There is some nuance where we might have to detect whether a value is an array or not, but that should rely on `component.multiple` rather than deriving it from untrusted value input.